src/unum/log/log_common.h

```diff
@@ -21,6 +21,19 @@
 // Max log file pathsname length the code will deal with
 #define LOG_MAX_PATH 128
 
+// A file containing the Prefix(directory)
+// All log files (For example unum.log)
+// will be prefixed with the content in this filename
+#ifdef PERSISTENT_FS_DIR_PATH
+#define LOGS_PREFIX_FILE PERSISTENT_FS_DIR_PATH"/unum_prefix_file.txt"
+#else
```

Comment:
please add
// PERSISTENT_FS_DIR_PATH
after #else and #endif
also 
why do we need #define LOGS_PREFIX_FILE "/etc/"? we should just leave it undefined and not even look for the file...

JFYI, eventually (hopefully soon) we'll get rid of that file thing altogether and just use config in persistent storage to enable/disable this and other debugging features.

Reply:
Added comments and removed the default paths.